src/jukebox/jukebox/playlistgenerator.py

```diff
@@ -191,7 +191,7 @@ def __init__(self, music_library_base_path='/'):
         but is omitted when generating the playlist entries. I.e. all files in the playlist are relative to this base dir
         """
         self.playlist = []
-        self._music_library_base_path = os.path.abspath(music_library_base_path)
+        self._music_library_base_path = os.path.abspath(os.path.expanduser(music_library_base_path))
```

Comment:
Will this work on the Pi as well? (I assume it does, just wanted to be sure)

Reply:
In the python doc (https://docs.python.org/3/library/os.path.html#os.path.expanduser) I see that this function has been in python pretty long and should be available in all used python versions. I tested successfully on my bananapi which currently runs Armbian 22.08.8 Jammy.